marketpulse/main/fxosprice_urls.py

```diff
@@ -4,6 +4,8 @@
 urlpatterns = patterns(
     '{0}.main.views'.format(settings.PROJECT_MODULE),
     url(r'^new/$', 'edit_fxosprice', name='edit_fxosprice'),
-    url(r'^contributions/$', 'all_fxosprice', name='all_fxosprice'),
+    url(r'^all/$', 'all_fxosprice', name='all_fxosprice'),
+    url(r'^(?P<user_pk>\d+)/contributions/$', 'all_fxosprice', name='user_fxosprice'),
```

Comment:
What if we changed this url to /contributions/me/? If we proceed with this, maybe we should consider changing the above url to /contributions/all. Thoughts?